### DISCUSSION TITLE
Loop Binding: Issue or 'User Error'?

### DISCUSSION THREAD

**Discussion by VikingDadMedic** [Budibase/budibase#6816] [Help]
I would appreciate a second set of eyes on extrapolating data from a loop (in automations). 

Specifically, after I run a query to return filtered rows, here are the results (there are 5 of them). 

![output-from-row-query](https://user-images.githubusercontent.com/91684138/179844525-b9658127-b7aa-467a-9b91-9320b2a5427c.png)

Now here is where I am running into a wall - the following step making an external query - using data points from above.

In the next step I want to loop thru the responses above and extract the primaryDisplay `value` (e.g., line 19: `"primaryDisplay": "8704686547618780400860562"` 

My sneaking suspicion is that despite the ingenious use of brackets `[Accounts Ref]` is likely the culprit - and/or nested data issue with a space issue. 

So how would one formulate the syntax of the binding to access the **rows ⇢ Account Ref ⇢ primaryDisplay ⇢ value** in order to bind it to the current item in the loop? 

![Syntax Expected](https://user-images.githubusercontent.com/91684138/179850369-5f8a9c04-515b-4a7c-80fd-94fb41282588.png)

**Comment by melohagan** [comment]
Hey @VikingDadMedic 

I believe the binding would be something like:

`Loop.currentItem[Accounts Ref].0.primaryDisplay`

or 

`Loop.currentItem[Accounts Ref][0][primaryDisplay]`
